Solutions/Neustar IP GeoPoint/Playbooks/Neustar-GetIPGeoInfo/azuredeploy.json

```diff
@@ -0,0 +1,371 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "metadata":  {
+        "title":  "Neustar-GetIPGeoInfo",
+        "description":  "When a new sentinel incident is created, this playbook gets triggered and performs the following actions: \n 1. Gets IP Addresses from incident. \n 2. Gets Geographical location information from Neustart IP GeoPoint API. \n 3. Summarize the details and add as a comment to the incident.",
+        "prerequisites":  
+                        "1. Prior to the deployment of this playbook, Neustar IP GeoPoint API Function App Connector needs to be deployed under the same subscription. \n 2. Refer to Neustar IP GeoPoint API Function App Connector documentation to obtain Neustar IP GeoPoint API Key and Shared Secret.",
+        "postDeployment":  [
+        ],
+        "prerequisitesDeployTemplateFile":  "",
```

Comment:
Add path to function app's azuredeploy.json file

Reply:
Added Function App Name as Parameter.